Calling/ClientApp/src/containers/GroupCall.ts

```diff
@@ -31,6 +31,7 @@ const mapStateToProps = (state: State, props: GroupCallProps) => ({
   },
   remoteParticipants: state.calls.remoteParticipants,
   streams: state.streams.streams,
+  threadId: state.thread.threadId,
```

Comment:
Is this used anywhere in `GroupCall`?

Reply:
oops, might have been left in while trying to debug something else.